### PR TITLE
Add test ensuring nginx redirect map points to valid urls

### LINKED ISSUE #1540: Create a script to validate the isomaping as part of CI/CD
make a script to check against the isomapping to validate the links.

Not sure if we need it after we move boost.io to boost.org, reassess when the time comes

### PR DESCRIPTION
**Pull request by GregKaleka**
Fixes #1540

### CODE REVIEW

**Review by rbbeeston — COMMENTED** [review]
I'm good with this if Dave is

**Comment by daveoconnor** [comment]
Would be good if this was part of the CI-CD runs on a merge to staging (would be enough, would cover merges to master too)

I have a script that tests the isomap spreadsheet contents. Should I add this? I'm thinking we probably want to have concrete and stable  tests on the examples that were compiled.

File: core/views.py [hunk 1]
```diff
@@ -560,7 +560,11 @@ class BaseRedirectView(View):
     @staticmethod
     def get_latest_library_version():
         """Return the latest version for a given library."""
-        return Version.objects.most_recent().stripped_boost_url_slug
+        most_recent_version = Version.objects.most_recent()
+        if most_recent_version is None:
+            # No versions imported - new dev setup or test
+            return "1.87.0"
```
Comment (daveoconnor) [inline core/views.py:566]: Is there another way to do this? This is bound to be forgotten after a future boost release and be opaque about why 1.87.0 is selected. I'd almost prefer if it just logged an error stating the user needs to update their libraries with an exception given it's just going to be for dev/test use.

**Comment by GregKaleka** [comment]
@daveoconnor if you have a script you'd like to add to a GHA, that's fine - happy to add it. I think the unit test I've added makes sense here too. Feel free to push to this branch if you agree.

**Comment by GregKaleka** [comment]
Merging this - @daveoconnor feel free to open another PR with a GHA script for this.

**Pull request merged**